nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/GetHDFS.java

```diff
@@ -220,16 +220,6 @@
                     .explanation(MIN_AGE.getName() + " cannot be greater than " + MAX_AGE.getName()).build());
         }
 
-        try {
```

Comment:
I'm not sure we should remove this, see this JIRA: https://issues.apache.org/jira/browse/NIFI-2553

Reply:
you're absolutely right, thanks! reverted this and added a unit test for the expected use case.